Charlie NTR #203 and #204

cbarcl01 opened the pull request:
Added new terms from issues #203 and #204 plus updated existing terms to include appropriate examples of usage or to update legacy definitions.

cbarcl01 committed:
updated "food object quality"

- updated "food object quality" by adding definition.

cbarcl01 committed:
added 'food packaging integrity'

- added 'food packaging integrity' with definition and example of usage.

cbarcl01 committed:
updated 'food packaging integrity' and 'food object quality'

- added term editor and dcdate curation.

cbarcl01 committed:
added 'food packaging intact'

- added 'food packaging intact' with definition and example of usage.

cbarcl01 committed:
added 'food packaging compromised'

added 'food packaging compromised' with definition and example of usage

cbarcl01 committed:
added 'food packaging opened by consumer'

- added 'food packaging opened by consumer' with definition and example of usage.

cbarcl01 committed:
added 'unpackaged or non-packaged food'

- added 'unpackaged or non-packaged food' with definition.

cbarcl01 committed:
added 'consumer-ready food packaging'

- added 'consumer-ready food packaging' with definition and definition source.

cbarcl01 committed:
added 'wholesale food bulk packaging'

- added 'wholesale food bulk packaging' with definition

cbarcl01 committed:
updated 'food packaging medium integrity'

- updated 'food packaging medium integrity' to ontological definition and added example of usage

cbarcl01 committed:
updated 'food packing medium compromised'

updated 'food packing medium compromised' example of usage.

cbarcl01 committed:
update 'food packaging'

- update 'food packaging' definition and definition source

cbarcl01 committed:
added 'unpackaged food'

added 'unpackaged food' with definition

cbarcl01 committed:
added 'non-packaged food'

- added 'non-packaged food' with definition.

cbarcl01 committed:
updated terms as 'ready to release'

- updated terms as 'ready to release' following review.

cbarcl01 committed:
updated 'food packing medium drained'

- added definition to 'food packing medium drained'

cbarcl01 committed:
updated 'food packing medium not drained'

- updated 'food packing medium not drained' with definition and example of usage

cbarcl01 committed:
updated 'food packing medium drained'

- updated 'food packing medium drained' with example of usage

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -13172,6 +13181,9 @@ AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002465 obo:IAO_0000122)
 
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002470 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00002470 "A physical object quality which can describe an attribute of a food item or related to a food product."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002470 "0000-0001-8008-8249")
```
When manually adding yourself as term editor you need to include the purl prefix. Protege does this for you automatically for new terms but not for every time you add the annotation.

e.g. use http://orcid.org/0000-0001-8008-8249 like you did in line 66605

```suggestion
AnnotationAssertion(dc:creator obo:FOODON_00002470 <http://orcid.org/0000-0001-8008-8249>)
```

cmrn-rhi replied:
I've noticed that outside of manual curation, we primarily use `term editor` [IAO_0000117] (e.g., in ROBOT tables) even for new terms. Going to check with Damion if he'd like us to only use `term editor` moving forward. If so we'll have to disable Protege's automated `dc:creator` annotation upon creation of new terms.

ddooley replied:
I think protege preferences -> "new entities metadata" allows you to control exactly which tags it adds.  I'm thinking like nico we should switch to dc:contributor !!

cmrn-rhi replied:
Right, forgot we could change it - so let's use `dc:contributor` as default and can discuss this further in Oc/Or to make sure everyone is on board.
![image](https://user-images.githubusercontent.com/48695054/172713147-52d308a9-d432-421e-92e9-a551b77e7f5f.png)

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -13230,7 +13242,7 @@ AnnotationAssertion(rdfs:label obo:FOODON_00002516 "gravlax"@en)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_00002517 "A can of chickpeas with a packing liquid that is colonized with a pathogen."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002517 obo:IAO_0000122)
 AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00002517 "A compromised food packing medium is a gaseous, liquid, or solid medium that has been exposed to the environment due to a failure in integrity of the food packaging.  The packaging failure may be intentional, e.g. opening a bag, or unintentional, e.g. a tear."@en)
 AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002517 "Damion Dooley"^^xsd:string)
```
Add a `term editor` annotation for yourself in addition to Damion's. 

```suggestion
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002517 "Damion Dooley"^^xsd:string)
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002517 <http://orcid.org/0000-0001-8008-8249>)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530218 "A can of peaches in syrup, in which the syrup is uncompromised."@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "A food object quality which describes the integrity of the medium in which the food is packed for preservation and handling or the medium surrounding foods."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Existing definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
```
When mentioning `existing` you are now referencing the current definition, not the retired one, as the context of it's application changed when the definition did. Recommend using "Previous" like you did in the heading of your comment.

```suggestion
AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Previous definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -58457,8 +58469,10 @@ SubClassOf(obo:FOODON_03480047 obo:FOODON_03480029)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "Type of container or wrapping defined by the main container material, the container form, and the material of the liner lids or ends. Also type of container or wrapping by form; prefer description by material first, then by form.  [FDA CFSAN 1995]"@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490100 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "A food contact material which encloses the food and can be in the type of a container, packaging or wrapping."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Existing definition was a legacy definition, not ontological and has been moved to comments."@en)
```
When mentioning `existing` you are now referencing the current definition, not the retired one, as the context of it's application changed when the definition did. Recommend using "Previous" like you did in the heading of your comment.
```suggestion
AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Previous definition was a legacy definition, not ontological and has been moved to comments."@en)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -58457,8 +58469,10 @@ SubClassOf(obo:FOODON_03480047 obo:FOODON_03480029)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "A food contact material which encloses the food and can be in the type of a container, packaging or wrapping."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Existing definition was a legacy definition, not ontological and has been moved to comments."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_03490100 "https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/food-packaging")
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03490100 "http://langual.org")
```
Add yourself to the edit history with a `term editor` annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03490100 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03490100 <http://orcid.org/0000-0001-8008-8249>)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "A food object quality which describes the integrity of the medium in which the food is packed for preservation and handling or the medium surrounding foods."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Existing definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530218 "http://langual.org")
```
Add yourself to the edit history with a `term editor` annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530218 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530218 <http://orcid.org/0000-0001-8008-8249>)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530219 "A can of chickpeas where the liquid medium has been drained."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530219 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530219 "A food object quality where the medium in which the food is packed is drained."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530219 "http://langual.org")
```
Add yourself to the edit history with a term editor annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530219 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530219 <http://orcid.org/0000-0001-8008-8249>)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -65876,7 +65896,10 @@ SubClassOf(obo:FOODON_03530219 obo:FOODON_03530218)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530220 "A can of chickpeas where the liquid medium has not been drained."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530220 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530220 "A food object quality where the medium in which the food is packed is still surrounding the food."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530220 "0000-0001-8008-8249"@en)
```
Add ORCiD PURL prefix and don't include an `@en` tag (probably an accidental carry over form a previous annotation).

```suggestion
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530220 <http://orcid.org/0000-0001-8008-8249>)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi commented on src/ontology/foodon-edit.owl:
```diff
@@ -66564,6 +66587,93 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03530295 "http://www.langual.o
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03601000 "A compromised plastic food wrapper."@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03601000 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03601000 "A food object quality which describes the stability and completeness of the packaging material."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03601000 "0000-0001-8008-8249")
```
Add ORCiD PURL prefix.

```suggestion
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03601000 <http://orcid.org/0000-0001-8008-8249>)
```

cmrn-rhi replied:
Resolved in later commits.

cmrn-rhi reviewed (changes requested):
Have recommended some small edits to do before passing it off to Damion.
Also waiting to hear back from Damion on exact preferences for `term editor` and `dc:creator` usage as that may have (minor) impact on a few annotations.

cbarcl01 committed:
changes requested from PR #209

changes implemented as requested from pull request #209

cbarcl01 committed:
added contributor details

- added Kayla Pennerman ORCiD as dc:contributor to terms generated from #203 and #204

cbarcl01 commented:
Changes made in Protege under a new commit and a new pull request generated.
